Sources/FlexLayout.swift

```diff
@@ -648,6 +648,73 @@ public final class Flex {
         return self
     }
     
+    /**
+      Set the left and right edges distance from the container edges in pixels.
+      This method is valid only when the item position is absolute (`view.flex.position(.absolute)`)
+      */
+    @discardableResult
+    public func horizontal(_ value: CGFloat) -> Flex {
+        yoga.left = YGValue(value)
+        yoga.right = YGValue(value)
+        return self
+     }
+
+     /**
+      Set the left and right edges distance from the container edges in percentage of its container width.
+      This method is valid only when the item position is absolute (`view.flex.position(.absolute)`)
+      */
+    @discardableResult
+    public func horizontal(_ percent: FPercent) -> Flex {
+        yoga.left = YGValue(value: Float(percent.value), unit: .percent)
+        yoga.right = YGValue(value: Float(percent.value), unit: .percent)
+        return self
+    }
+    
+    /**
+     Set the start and end  edges (LTR=right, RTL=left) distance from the container edges in pixels.
+     This method is valid only when the item position is absolute (`view.flex.position(.absolute)`)
+     */
+    @discardableResult
+    public func horizontalDirected(_ value: CGFloat) -> Flex {
```

Comment:
Not sure that `horizontalDirected` is required since we set `start` and `end` to the same value, this is equivalent to setting the `left` and `right`.

Reply:
Yeap, you are right, miss this moment. Will remove `horizontalDirected `